infra/lib/env-config.ts

```diff
@@ -39,6 +39,16 @@ export type EnvConfig = {
   usageReportUrl?: string;
   fhirServerUrl?: string;
   systemRootOID: string;
+  cwOrgName: string;
+  cwMemberOid: string;
+  cwOrgManagementKey: string;
+  cwOrgManagementCertification: string;
+  cwOrgMemberKey: string;
+  cwOrgMemberCertificate: string;
+  gatewayEndpointLocation: string;
+  gatewayAuthorizationServerEndpoint: string;
+  gatewayAuthorizationClientId: string;
+  gatewayAuthorizationClientSecret: string;
 } & (
```

Comment:
Ill need to add this to the example or make them optional.Realized in the build error